pkg/webhook/v1/manifestwork_validating.go

```diff
@@ -76,16 +76,24 @@ func (r *ManifestWorkWebhook) validateRequest(newWork, oldWork *workv1.ManifestW
 }
 
 func validateManifests(work *workv1.ManifestWork) error {
+	return ValidateManifestWorkSpec(work.Spec)
+}
+
+func ValidateManifestWorkSpec(manifestWorkSpec workv1.ManifestWorkSpec) error {
```

Comment:
nit the name of ValidateManifestWorkSpec is a bit confusing, we actually validate manifest in the spec.

Reply:
I name it after the given param manifestWorkSpec and there is a validateManifest func [Line 106](https://github.com/open-cluster-management-io/work/pull/182/files#diff-c05eb54e7b9381bee7827595cf0c481f6c38395bdafaccfb692136bfddba67aaR106)

Reply:
I would remove validateManifests in line 78 and name this func as ValidateManifests. Will it be OK.

Reply:
okay, done